fedimint-core/src/api.rs

```diff
@@ -613,6 +619,54 @@ impl WsClientConnectInfo {
     }
 }
 
+impl FromStr for WsClientConnectInfo {
+    type Err = anyhow::Error;
+
+    fn from_str(encoded: &str) -> Result<Self, Self::Err> {
```

Comment:
can we add the expected representation as documentation somewhere? I think it'd be nice to have it

```
[ id (48 bytes) ] ([ url len (2 bytes) ] [ url bytes (url len bytes) ])+
```